docs/typescript/how-to-handle-a-query-in-a-workflow-in-typescript.md

```diff
@@ -0,0 +1,31 @@
+---
+id: how-to-handle-a-query-in-a-workflow-in-typescript
+title: How to handle a Query in a Workflow in TypeScript
+sidebar_label: Handle Query
+description: To handle a Query in a Workflow, define a Query handler method using the `handle.query(query, ...args)` annotation in the Workflow interface.
+tags:
+  - typescript
+  - developer-guide
+---
+
+Query Handlers can return values inside a Workflow in TypeScript.
+
+You make a Query with `handle.query(query, ...args)`. A Query needs a return value, but can also take arguments.
+
+```typescript
+import * as wf from '@temporalio/workflow';
+
+function useState<T = any>(name: string, initialValue: T) {
```

Comment:
Not a good example. Here's one:

https://docs.temporal.io/typescript/workflows#define-signals-and-queries-statically